README.md

```diff
@@ -1,118 +1,145 @@
-	background-size: cover;
+```scss
+.icon {
+	@include oIconsBaseStyles;
 }
 ```
 
-We provide a mixin to make this easier which is documented in Sassdoc [http://registry.origami.ft.com/components/o-ft-icons#docs-css](http://registry.origami.ft.com/components/o-ft-icons#docs-css).
+Which outputs:
 
-----
+```scss
+.icon {
+	display: inline-block;
+	width: 128px;
+	height: 128px;
+	background-repeat: no-repeat;
+	background-size: contain;
+	background-position: 50%;
+	background-color: transparent;
+	vertical-align: middle;
+}
+```
 
-## Add / edit icons, build the web font and demo page
+### 3. Manually using the [responsive image service](https://image.webservices.ft.com)
+
+If you can't use the mixins, and you need to see a custom size or colour, you can also use the [responsive image service](https://image.webservices.ft.com) to fetch the icons in a very similar way as to how the mixin works:
+
+```scss
+element {
+    display: inline-block;
+    width: 100px;
+    height: 100px;
 
-**warning** Fontforge is notoriously flaky at the moment. We are planning to deprecate its use shortly so if you need to make a minor amendment and these steps don't work for you first time, it might be worth getting someone who has fontforge working on their machine already to make the change. If you really want to try and get this running on your machine and it's giving you trouble, a gist of the steps I tried is here: https://gist.github.com/alicebartlett/1e785aa6a9baa8876d3f
+    // Older browsers: PNG fallback (resized to 100px wide)
+    background-image: url('//image.webservices.ft.com/v1/images/raw/fticon:tick?width=100&format=png&source=my-product');
 
-1. Install the following:
+    // Modern browsers: SVG covering the whole size of the element
+    // we declare multiple backgrounds so that only modern browsers read this property
+    background-image: url('//image.webservices.ft.com/v1/images/raw/fticon:tick?format=svg&source=my-product'), none;
+    background-size: cover;
+}
+```
 
-	* [fontforge](http://fontforge.org/)
-	* [ttfautohint](http://www.freetype.org/ttfautohint/#download)
-	* [X11](http://support.apple.com/kb/ht5293) (Mac only)
+## Add / edit icons, build the  demo page
 
-2. Clone the repository and install dependencies:
+1. Clone the repository and install dependencies:
 
-		git clone https://github.com/Financial-Times/o-ft-icons.git
-		cd o-ft-icons
+		git clone https://github.com/Financial-Times/o-icons.git 
+		cd o-icons
 		obt install
 
-3. Add or edit an SVG file to the `svg` folder (see [SVG file naming rules](#svg-file-naming-rules)).
-4. Generate the web font from the SVG sources:
+1. Add or edit an SVG file to the `svg` folder (see [SVG file naming rules](#svg-file-naming-rules)).
+1. The `_icon-list.scss` file is generated by a Gulp task. Also, the `data.json` file used for demos is generated by a Gulp task. When adding a new icon, these files need to be rebuilt. You just need to run:
```

Comment:
Could you say: "The `_icon-list.scss` and `data.json` files are generated by a Gulp task. When adding a new..."